### PR TITLE
[PATCH v2] Fix l2fwd test application termination

### PR DESCRIPTION
**Pull request by matiaselo**

### CODE REVIEW

File: test/common_plat/performance/odp_l2fwd.c [hunk 1]
```diff
@@ -400,6 +407,22 @@ static int run_worker_sched_mode(void *arg)
 	/* Make sure that latest stat writes are visible to other threads */
 	odp_mb_full();
 
+	/* Wait that pktio devices are stopped */
```
Comment (Bill-Fischofer-Linaro) [inline test/common_plat/performance/odp_l2fwd.c]: Wait until?

File: test/common_plat/performance/odp_l2fwd.c [hunk 2]
```diff
@@ -502,6 +525,27 @@ static int run_worker_plain_queue_mode(void *arg)
...
```
Comment (Bill-Fischofer-Linaro) [inline test/common_plat/performance/odp_l2fwd.c]: Wait until...

Comment (matiaselo) [inline test/common_plat/performance/odp_l2fwd.c]: Thanks, will fix.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**